### PR TITLE
Feature/gramine

### PR DESCRIPTION
**Pull request by james-toussaint**

### CODE REVIEW

File: for-developers/confidential-computing/create-your-first-gramine-app.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Build a Gramine application
+
+Lorem ipsum
+
+
+
+You just built and run your Gramine application, now you can go a step further with:
```
Comment (jbern0rd) [inline for-developers/confidential-computing/create-your-first-gramine-app.md]:
```suggestion
You just built and ran your Gramine application, now you can go a step further with:
```
or maybe just copy and paste the **Next step** section from the **Build a Scone TEE app** page

**Review by jbern0rd — COMMENTED** [review]
To check with Pierre, **Prettier** seems to recommend `-` instead of `*` for unordered lists.

File: for-developers/confidential-computing/create-your-first-gramine-app.md (hunk 1 above)
Comment (james-toussaint) [reply for-developers/confidential-computing/create-your-first-gramine-app.md]: Thank you: https://github.com/iExecBlockchainComputing/documentation/pull/113/commits/de3f731f43d2afee68fb261d2494018944d9bfc8

File: SUMMARY.md [hunk 2]
```diff
@@ -1,39 +1,43 @@
...
+- [Proof of Contribution](key-concepts/proof-of-contribution.md)
+- [Pay per task model](key-concepts/pay-per-task-model.md)
 
 ## Help
 
-* [v6 to v7 migration](help/migration.md)
-* [Glossary](help/glossary.md)
-* [Assets](help/assets.md)
-* [Contact Us](help/contact-us.md)
+- [v6 to v7 migration](help/migration.md)
```
Comment (jbern0rd) [inline SUMMARY.md]: To discuss with Pierre who changes this to avoid missing the update ?
```suggestion
- [v7 to v8 migration](help/migration.md)
```

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**